src/objective/variable_om_costs.jl

```diff
@@ -30,6 +30,7 @@ function variable_om_costs(m::Model, t1)
         expr_sum(
             +unit_flow[u, n, d, s, t] *
             duration(t) *
+            sum(weight(temporal_block=blk) for blk in t.blocks) *
```

Comment:
I have a small suggestion: replace `t.blocks` by `blocks(t)`. It's more api-friendly than accessing fields directly.

Reply:
Yes that looks better. Also, related to the original MR message:
> currently the temporal_block weights are summed up. This is a bug and needs to be changed to prod(...) . Currently this is not supported by Spineopt and interface
I have tried to use prod(..) or Iterators.prod(..) here, but the iteration method seems to fail here. Do you have an idea on that? I guess alternatively reduce(*,...) could work